DuckDuckGo/NetworkProtectionVPNSettingsView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  NetworkProtectionVPNSettingsView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#if NETWORK_PROTECTION
+
+import SwiftUI
+
+@available(iOS 15, *)
+struct NetworkProtectionVPNSettingsView: View {
+
+    var body: some View {
+        List {
+            toggleSection(
+                text: UserText.netPAlwaysOnSettingTitle,
+                footerText: UserText.netPAlwaysOnSettingFooter
+            )
+            toggleSection(
+                text: UserText.netPSecureDNSSettingTitle,
+                footerText: UserText.netPSecureDNSSettingFooter
```

Comment:
The designs have a “Learn more” link, but we’re not including that just yet. https://app.asana.com/0/0/1204630981406975/1205756069414403/f